data-access-analysis-tools/index-portal-intro.rst

```diff
@@ -119,9 +121,69 @@ Note that corresponding plot point for the selected row in the table is differen
 
     An updated results view in which the xy plot uses the magnitude columns.
 
-See also the :ref:`Examples-DP0-1-Notebooks` for additional demonstrations of how to use the Portal's Single Table Query.
+See also for additional demonstrations of how to use the Portal's Single Table Query.:ref:`Examples-DP0-1-Notebooks`
```

Comment:
The phrasing of this sentence sounds strange. Also, the reference isn't rendering correctly at https://dp0-1.lsst.io/v/PREOPS-528/data-access-analysis-tools/index-portal-intro.html